src/app/content/components/__snapshots__/Content.spec.tsx.snap

```diff
@@ -3,21 +3,21 @@
 exports[`content matches snapshot 1`] = `
 Array [
   <a
-    className="sc-bdVaJa RfHPw"
+    className="css-1axmsgr e1wgaj6u0"
```

Comment:
i wonder is these class names suffer from the same issue as styled where they will all update anytime any styles changes? 

also it looks like you branched off before i added the jest-snapshot thing, is there a comparable thing for emotion that will show the styles in the snapshot?